rtl/README.md

```diff
@@ -20,18 +20,84 @@ yarn add @pxblue/storybook-rtl-addon
 In your `main.js` file, register this addon:
 
 ```
-addons: [
-    '@pxblue/storybook-rtl-addon/register',
-]
+addons: ['@pxblue/storybook-rtl-addon/register']
 ```            
 
-> **Angular Components**: If your component uses the Direction service, use the exported function `getDirection()` to supply the `[dir]` directive with the appropriate direction. 
+## Usage
+
+This package exports two functions `getDirection` & `useDirection` that returns the current `Direction` ('rtl' or 'ltr').
+
+### getDirection
+`getDirection` is used within Stories to access the current `Direction`.
+
+For detailed usage for the `getDirection` function, see the Examples section further below.
+
+> **Angular Components**: If your component uses the Direction service, use the exported function `getDirection()` to supply the `[dir]` directive with the appropriate direction.
+
+### useDirection
+`useDirection` is a React hook that emits any `Direction` change and can be used inside of Storybook Decorators.
```

Comment:
```suggestion
`useDirection` is a React hook that returns the current `Direction` and re-emits on every `Direction` change (when toggling the RTL sidebar button). It can be used to e.g. set values for Providers in a Decorators or Stories.
```

`useDirection` can technically be used everywhere in Storybook, not just Decorators.